src/apps/appstore/i18n/en-US.json

```diff
@@ -72,11 +72,11 @@
 			"cancel": "Cancel",
 			"actionBtn": "Disconnect from Marketplace",
 			"actionKey": "DISCONNECT",
-			"messageIntro": "This action CANNOT be undone. This will permanently disassociate your cluster from 2600Hz Marketplace and remove any apps or services purchased from marketplace.",
+			"messageIntro": "This action CANNOT be undone. This will permanently disassociate your cluster from the 2600Hz Marketplace and remove any apps or services purchased from marketplace.",
 			"messageExplanation": "Please type the word \"{{variable}}\" in the following input and then click on the Disconnect button to proceed."
 		},
 		"dangerDisableDialog": {
-			"content": "This will disable Marketplace connector. Next time your Kazoo node starts, the apps purchased through marketplace will not be started! Do not forget to update the list startup apps."
+			"content": "This will disable the Marketplace connector. The next time your KAZOO nodes start, the apps purchased through marketplace will not be started. Do not forget to remove those applications from your start up list."
```

Comment:
Should we provide instructions in App Marketplace or here to disable the app from the startup list?

Reply:
`The next time your KAZOO nodes start` should it say `The next time your KAZOO nodes starts`  not sure.

Reply:
There are multiple nodes, and `start` is used as verb here (only third-person gets conjugated to `starts`).

Reply:
> Should we provide instructions in App Marketplace or here to disable the app from the startup list?

@icehess ^^